frame/staking/src/pallet/mod.rs

```diff
@@ -1038,7 +1044,9 @@ pub mod pallet {
 						}
 					})
 				})
-				.collect::<result::Result<Vec<T::AccountId>, _>>()?;
+				.collect::<result::Result<Vec<T::AccountId>, _>>()?
+				.try_into()
+				.expect("bound checked in previous line; qed");
```

Comment:
```suggestion
				.expect("bound checked above; qed");
```

slightly more correct I think :P